### ISSUE TITLE
feat: use monorepo preset

### ISSUE THREAD

**Issue by ecklf** [preset/preset#34]
It would be nice to use a preset located in a monorepo with the GitHub URL syntax

```sh
npx use-preset username/repository/tree/branch/package
```

Alternatives:
```sh
# Omit "tree"
npx use-preset username/repository/branch/package
# Branchname will default to master | main
npx use-preset username/repository/package
```

**Comment by innocenzi** [comment]
This is now implemented in [v0.1.11](https://github.com/use-preset/use-preset/releases/tag/v0.1.11). See #38. ^-^

**Comment by jrson83** [comment]
What would be the proper way to have short aliases when having presets in a monorepo in nested folder `./packages/presets/`:

```
.
└── packages/
    └── presets/
        └── templates/
            ├── react/
            │   ├── js/
            │   └── ts/
            ├── vue/
            │   ├── js/
            │   └── ts/
            └── preset.ts
```

Or as an alternative approach:

```
.
└── packages/
    └── presets/
        ├── react/
        │   ├── templates/
        │   │   ├── js/
        │   │   └── ts/
        │   └── preset.ts
        └── vue/
            ├── templates/
            │   ├── js/
            │   └── ts/
            └── preset.ts
```

How can I apply a preset without having to specify `packages/presets` in the command flag?
